app/helpers/action_links_helper.rb

```diff
@@ -22,6 +22,23 @@ def renew_link_for(resource)
     ad_privacy_policy_path(reg_identifier: resource.reg_identifier)
   end
 
+  def display_renewal_link_for?(resource)
```

Comment:
Can we give this method a more specific name? I feel like `display_renewal_link_for` and `display_renew_link_for` are too similar! Maybe specify this is the magic link?

Reply:
Perfectly fair point. I was taking the name from the title of the new section to try and make that link obvious. Bloomin naming things is 💩